grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/KotlinGrpcClientStubFactory.java

```diff
@@ -62,8 +63,8 @@ public ServiceDescriptor findServiceDescriptor(Class<?> clientType) {
     private static Annotation stubForAnnotation(Class<?> clientType) {
         try {
             @SuppressWarnings("unchecked")
-            final Class<Annotation> annotationClass =
-                    (Class<Annotation>) Class.forName("io.grpc.kotlin.StubFor");
+            final Class<Annotation> annotationClass = (Class<Annotation>) Class.forName(
+                    BindableService.class.getPackage().getName() + ".kotlin.StubFor");
```

Comment:
How about fallback to `io.grpc.kotlin.StubFor` just in case a user only shades gRPC-Java?

Reply:
I guess then we will have a problem if a user had a separate non-shaded gRPC-Java and gRPC-Kotlin in the classpath while shading gRPC-Java and gRPC-Kotlin linked with Armeria, i.e. a user wants to keep two gRPC-Java/Kotlin versions.

Reply:
That makes sense.